services/ux-backend/handlers/rotatekeys/handler.go

```diff
@@ -0,0 +1,90 @@
+package rotatekeys
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"os"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
```

Comment:
Can we declare this constant globally since it will be needed in multiple handlers?

Reply:
Maybe in a `utils.go` or `constants.go` file directly under the `handlers` package

Reply:
done.